lib/src/license_detection/primary_filter.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:pana/src/license_detection/license.dart';
+
+/// Filters the corpus licenses and
+/// returns a list of [License] which might
+/// be a possible match.
+List<License> filter(Map<String, int> occurences, List<License> knownLicenses) {
```

Comment:
```suggestion
List<License> filter(Map<String, int> occurrences, List<License> possibleLicenses) {
```

Reply:
Should it be `possibleLicenses` because we the filtering stage we are passing all the known licenses aren't we?